src/style.css

```diff
@@ -1,3 +1,87 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+:root {
+  --active-link-color: #00acdc;
+}
+
+a {
+  display: inline-block;
+  padding: 0;
+  margin: 0;
+  line-height: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for Roboto to ensure text is displayed with a sans-serif font even if Roboto fails to load.